app/services/workarea/copy_order.decorator

```diff
@@ -0,0 +1,10 @@
+module Workarea
+  decorate CopyOrder, with: :package_products do
+    # When an order is copied, we don't want the bundled items in the items
+    # collection yet. They will still exist in #bundled_items.
```

Comment:
I assume they still get created... somewhere?

Reply:
bundled items is its own embedded collection. In cart and checkout they live only in that separate collection. Copied orders will have them in both `bundled_items`, and `items`. Taking them out of `items` ensures pricing and such still works as expected should the order change before being placed.